universal-login-relayer/migrations/20190528140000_create_messages.js

```diff
@@ -0,0 +1,18 @@
+/*
+  THIS IS A MIGRATION
+  This file SHOULD NOT be RENAMED or CHANGED in ANY WAY!
+  If you need to change the database schema create a new migration.
+*/
+
+exports.up = async (knex) => {
```

Comment:
I guess it is fine for now. In the future, I would create to tables in one migration